Content.Shared/CCVar/CCVars.Atmos.cs

```diff
@@ -150,5 +150,5 @@ public sealed partial class CCVars
     ///    Adjusted for STARLIGHT use
     /// </summary>
     public static readonly CVarDef<float> AtmosTankFragment =
-        CVarDef.Create("atmos.max_explosion_range", 10f, CVar.SERVERONLY);
+        CVarDef.Create("atmos.max_explosion_range", 15f, CVar.SERVERONLY);
```

Comment:
starlight comment, I know it was originally missing here to begin with but it should be added now since we are touching it again